ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v1/checkboxgroup/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,160 @@
+                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <items jcr:primaryType="nt:unstructured">
+                            <placeholder
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:hideResource="{Boolean}true"/>
+                            <type
+                                    wrapperClass="cmp-adaptiveform-checkboxgroup__type"
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/select"
+                                    sling:orderBefore="enum"
+                                    fieldLabel="Data Type of the Value"
+                                    name="./type"
+                                    key="String"
+                                    value="string[]">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <string jcr:primaryType="nt:unstructured" text="String" value="string[]"/>
+                                    <boolean jcr:primaryType="nt:unstructured" text="Boolean" value="boolean[]"/>
+                                    <number jcr:primaryType="nt:unstructured" text="Number" value="number[]"/>
+                                </items>
+                            </type>
+                            <enum
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/multifield"
+                                    fieldDescription="Enter Enum Values"
+                                    fieldLabel="Enum">
+                                <field
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                        name="./enum"
+                                        required="{Boolean}false"/>
+                            </enum>
+                            <enumNames
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="fd/af/authoring/components/granite/form/formsmultifield"
+                                    fieldDescription="Enter values to be displayed."
+                                    fieldLabel="Enum Names">
+                                <field
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                        name="./enumNames"
+                                        required="{Boolean}false"/>
+                            </enumNames>
```

Comment:
we are capturing them separately

Reply:
For now, we are taking separate, Since Radio button has already done this change. Will incorporate them from the base after those are available.